### PR TITLE
Correct response information for events API POST and PUT endpoints

### LINKED ISSUE #2459: POST & PUT requests to events API won't return a response body
### Description
Context: https://sensucommunity.slack.com/archives/C9BB9AW7K/p1589487879288700

These requests will not return a response body, which is the intended behavior.

### Category

This is an issue with:

- [x] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
- https://docs.sensu.io/sensu-go/latest/api/events/#eventsentitycheck-post
- https://docs.sensu.io/sensu-go/latest/api/events/#eventsentitycheck-put
- ???

### Corrections needed
Only a 204 response code is returned, with an empty body.

### Additional context

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
For POST and PUT endpoints in the events API doc:
- Add HTTP 201 Created response in code examples
- Remove incorrect notes about HTTP 200 OK and copy of created/updated resource in responses
- Correct "Success" HTTP response codes to 201 (Created) in endpoint description tables

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2459

When approved, I will propagate back to 5.17.

### CODE REVIEW

File: content/sensu-go/5.20/api/events.md [hunk 1]
```diff
@@ -212,8 +212,8 @@ curl -X POST \
 }' \
 http://127.0.0.1:8080/api/core/v2/namespaces/default/events
 
-HTTP/1.1 200 OK
-{"timestamp":1552582569,"id":"caaf2c38-2afb-4f96-89b3-8ca5c3e6f449","entity":{"entity_class":"proxy","system":{"network":{"interfaces":null}},"subscriptions":null,"last_seen":0,"deregister":false,"deregistration":{},"metadata":{"name":"server1","namespace":"default"}},"check":{"handlers":["slack"],"high_flap_threshold":0,"interval":60,"low_flap_threshold":0,"publish":false,"runtime_assets":null,"subscriptions":[],"proxy_entity_name":"","check_hooks":null,"stdin":false,"subdue":null,"ttl":0,"timeout":0,"round_robin":false,"executed":0,"history":null,"issued":0,"output":"Server error","state":"failing","status":2,"total_state_change":0,"last_ok":0,"occurrences":0,"occurrences_watermark":0,"output_metric_format":"","output_metric_handlers":null,"env_vars":null,"metadata":{"name":"server-health"}},"metadata":{}}
+
+HTTP/1.1 201 Created
```
Comment (palourde) [inline content/sensu-go/5.20/api/events.md:216]: I just realized we are missing a `\` at the end of line 193 above!

E.g. It should be `-H 'Content-Type: application/json' \`

File: content/sensu-go/5.20/api/events.md [hunk 2]
```diff
@@ -717,20 +713,16 @@ curl -X PUT \
   }
 }' \
 http://127.0.0.1:8080/api/core/v2/namespaces/default/events/server1/server-health
-{{< /highlight >}}
 
-The request returns an HTTP `200 OK` response and the resulting event definition.
+
+HTTP/1.1 201 Created
```
Comment (palourde) [inline content/sensu-go/5.20/api/events.md:718]: Same as above, but for the new line 696

File: content/sensu-go/5.20/api/events.md [hunk 3]
```diff
@@ -629,20 +629,16 @@ curl -X POST \
...
```
Comment (palourde) [inline content/sensu-go/5.20/api/events.md:634]: Same thing on line 612!

**Review by palourde — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.20/api/events.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.20/api/events.md:216]: Thank you

**Comment by hillaryfraley** [comment]
Thank you for catching this, @palourde . I found a couple more missing and added them.

**Review by palourde — APPROVED** [review]

**Pull request merged**